src/main/resources/default/taglib/k/toc.html.pasta

```diff
@@ -0,0 +1,9 @@
+<i:pragma name="description">
+    Renders a table of content uniformly.
+</i:pragma>
+
+<k:section heading="Inhaltsverzeichnis" anchor="toc">
```

Comment:
Inhaltsverzeichnis - in german?

Reply:
Hmm ja da hast du recht... muss mal überlegen. =)